hieradata/integration.yaml

```diff
@@ -243,6 +243,8 @@ monitoring::checks::enable_support_check: false
 monitoring::checks::pingdom::enable: false
 monitoring::checks::ses::region: eu-west-1
 monitoring::checks::smokey::environment: 'integration'
+monitoring::uptime_collector::enabled: true
```

Comment:
if this is `true` everywhere can we just set that to the default?

Reply:
Oh yes... in my mind this was here to stop the uptime collecting service going on anything other than the monitoring box but I realise that doesn't make sense.